src/dotnet/App.Maui/Platforms/Android/Notifications/FirebaseMessagingService.cs

```diff
@@ -95,66 +80,66 @@ public override void OnMessageReceived(RemoteMessage message)
             }
         }
 
-        ShowNotification(title, text, imageUrl, message.Data);
+        ShowChatMessageNotification(data);
     }
 
     // Private methods
-
-    private void ShowNotification(
-        string title,
-        string text,
-        string? imageUrl,
-        IDictionary<string, string> data)
+    private static bool ShowGetAttentionNotification(
+        NotificationData data,
+        long messageSentTime)
     {
-        data.TryGetValue("tag", out var tag);
-        var intent = new Intent(this, typeof(MainActivity));
-        intent.AddFlags(ActivityFlags.SingleTop);
-
-        foreach (var key in data.Keys) {
-            string value = data[key];
-            intent.PutExtra(key, value);
+        var chatId = data.ChatId;
+        if (chatId.IsNone) {
+            Log.LogWarning("Can't show get-attention notification. Invalid ChatId. Ref messageId: '{MessageId}'", data.MessageId);
+            return false;
         }
-
-        if (Log.IsEnabled(LogLevel.Debug)) {
-            var dataAsText = data.Select(c => $"'{c.Key}':'{c.Value}'").ToCommaPhrase();
-            Log.LogDebug("About to show ShowNotification '{Text}'. Data: {Data}", text, dataAsText);
+        var sentTime = new Moment(messageSentTime * 10_000).ToDateTime();
+        var title = data.Title ?? "";
+        var separatorIndex = title.IndexOf('@');
+        if (separatorIndex >= 0) {
+            // var part1 = title.Substring(0, separatorIndex).Trim();
+            var part2 = title.Substring(separatorIndex + 1).Trim();
+            title = part2;
```

Comment:
why not 
`title = title.Substring(separatorIndex + 1).Trim()`

Reply:
done